.eslintrc.cjs

```diff
@@ -5,7 +5,7 @@ module.exports = {
     root: true,
     rules: {
         "@typescript-eslint/no-var-requires": "warn",
-        "@typescript-eslint/no-extra-semi": "warn",
+        "@typescript-eslint/no-extra-semi": "error",

```

Comment:
Just remove the entire line. `warn` was an override. Setting it to `error` is letting it go back to the default.